symplyphysics/laws/gravity/first_cosmic_velocity.py

```diff
@@ -0,0 +1,34 @@
+from sympy import Eq, solve, sqrt
```

Comment:
Avoid marketing names for laws. Rename for people who don't know what is first cosmic velocity.

Reply:
I don't know how to call it, because there are 4 cosmic velocities and all named same.